readme.md

```diff
@@ -71,6 +71,13 @@ Returns an `Object` with the result, or `undefined` if there is no active window
 - `url` *(string?)* - URL of the active browser tab if the active window is Safari, Chrome, Edge, or Brave *(macOS only)*
 - `memoryUsage` *(number)* - Memory usage by the window owner process
 
+### activeWin.isAccessGranted()
+
+Checks if the app has enough access to get the active window
```

Comment:
It's not really the app though, it's the CLI tool. It's just that the CLI tool inherits the app's permissions (I think).